### ISSUE TITLE
Network Up/Down information should have dynamic spacing

### ISSUE THREAD

**Issue by codebytere** [exelban/stats#92]
**Describe the bug**

The Network `NSMenuItem` is intended to account for what looks to be a threeish-digit max (image pulled from settings): 
<img width="75" alt="Screen Shot 2020-09-23 at 6 57 06 PM" src="https://user-images.githubusercontent.com/2036040/94092038-96019d80-fdce-11ea-8758-eba2ca360785.png">

But does not shrink or expand to account for the current numeric value, leaving perhaps too much whitespace at lower values (image from my toolbar):
<img width="75" alt="Screen Shot 2020-09-23 at 6 57 01 PM" src="https://user-images.githubusercontent.com/2036040/94092176-e678fb00-fdce-11ea-8eca-e234ba49d622.png">

It'd be awesome if the dots/characters/etc accounted for this so as not to create quite as much whitespace.

**Details:**
 - Macbook Pro 2019
 - MacOS Catalina 10.15.6
 - Stats v2.3.3

**Comment by exelban** [comment]
Hi.

TL;DR static size of this widget allows ensuring the static position of elements in the menu bar.

There is one problem with this approach. Network value is too dynamic. I mean that value (upload/download) can change each second. And if widget size will be dynamic (depends on text size), the widget will be jumping. For example, if youtube (or some stream) is opened, the download value can jump from 0 KB/s to 10.3 MB/s. And if the network widget is not first in a row, your all menu bar will be jumping when the network is active (because the width of the widget change).

**Comment by codebytere** [comment]
Thanks for the background! that does make a lot of sense, and either way it's a very small deal in the big scheme of things :) Thanks for your work on this app, and feel free to close this issue out as you see fit.
